rentals.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Next we modify the constructor to add the rental object to the respective arrays of the person and book objects.

  ```
  def initialize(date, book, person)
    @date = date

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
  ```

Reply:
Fixed! 🔨

Reply:
This change has been done well